run-vmtest/run.sh

```diff
@@ -46,12 +46,11 @@ foldable start vmtest "Starting virtual machine..."
 
 # Tests may be comma-separated. vmtest_selftest expect them to come from CLI space-separated.
 T=$(echo ${KERNEL_TEST} | tr -s ',' ' ')
-# HACK: We need to unmount /tmp to access /tmp from the container....
-vmtest -k "${VMLINUZ}" --kargs "panic=-1 sysctl.vm.panic_on_oom=1" "umount /tmp && \
```

Comment:
I was surprised to see this.... but we essentially do not need to umount /tmp since https://github.com/danobi/vmtest/commit/0a63dce0e073273ba79c8dd4bfba10b571bcbd09

Reply:
Yeah. I actually **had to** remove it, otherwise it failed to umount.